packages/orchestration/src/exos/chain-hub.js

```diff
@@ -54,7 +54,38 @@ export const connectionKey = (chainId1, chainId2) => {
   return [chainId1, chainId2].sort().join(CHAIN_ID_SEPARATOR);
 };
 
+const ChainIdArgShape = M.or(
```

Comment:
consider:
```suggestion
/** accepts `chainId` or `ChainAddress` object */
const ChainIdArgShape = M.or(
```

Reply:
good idea. I'll make a note to get that into a future PR